fs/expose/dokan/__init__.py

```diff
@@ -1034,29 +1067,29 @@ class Win32SafetyFS(WrapFS):
 
     """
 
-    def __init__(self,wrapped_fs,allow_autorun=False):
+    def __init__(self, wrapped_fs, allow_autorun=False):
         self.allow_autorun = allow_autorun
-        super(Win32SafetyFS,self).__init__(wrapped_fs)
+        super(Win32SafetyFS, self).__init__(wrapped_fs)
 
-    def _encode(self,path):
-        path = relpath(normpath(path))
-        path = path.replace(":","__colon__")
+    def _encode(self, path):
+        path = relpath(normpath(path.replace('\\', '/')))
```

Comment:
Given that `Win32SafetyFS` is a `WrapFS`, IIRC all the paths it operates on should already be pyfs-paths? The slash-conversion should be contained only in the Dokan-code?